app/components/formatted_time.test.js

```diff
@@ -18,70 +18,69 @@ describe('FormattedTime', () => {
     it('should render correctly', () => {
         console.error = jest.fn();
 
-        let wrapper = renderWithIntl(
+        const viewOne = renderWithIntl(
```

Comment:
Instead of calling `wrapper`, it's called `view` (see blog post on why) since it's actually a view or actual rendering of a component instead of being wrapped.